### ISSUE TITLE
viewport user-scalable setting accessibility violation

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#5029]
A change made in RStudio 1.2 to prevent double-tap zoom on Mac is a WCAG level AA accessibility violation flagged by automated tests.

https://github.com/rstudio/rstudio/issues/2396
https://github.com/rstudio/rstudio/pull/2467

Background: https://dequeuniversity.com/rules/axe/3.2/meta-viewport

Logging this as we'll either need to call this out as an exception, or make this a desktop-only thing
